[2022-11-18T04:30:32.497+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: spark_operator.spark-job manual__2022-11-18T04:30:31.093548+00:00 [queued]>
[2022-11-18T04:30:32.504+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: spark_operator.spark-job manual__2022-11-18T04:30:31.093548+00:00 [queued]>
[2022-11-18T04:30:32.505+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-18T04:30:32.505+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-18T04:30:32.506+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-18T04:30:32.515+0000] {taskinstance.py:1383} INFO - Executing <Task(SparkSubmitOperator): spark-job> on 2022-11-18 04:30:31.093548+00:00
[2022-11-18T04:30:32.519+0000] {standard_task_runner.py:55} INFO - Started process 77 to run task
[2022-11-18T04:30:32.521+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'spark_operator', 'spark-job', 'manual__2022-11-18T04:30:31.093548+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/load_data_to_spark.py', '--cfg-path', '/tmp/tmpnirnbj2a']
[2022-11-18T04:30:32.522+0000] {standard_task_runner.py:83} INFO - Job 35: Subtask spark-job
[2022-11-18T04:30:32.561+0000] {task_command.py:376} INFO - Running <TaskInstance: spark_operator.spark-job manual__2022-11-18T04:30:31.093548+00:00 [running]> on host 69dc550e7316
[2022-11-18T04:30:32.604+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=spark_operator
AIRFLOW_CTX_TASK_ID=spark-job
AIRFLOW_CTX_EXECUTION_DATE=2022-11-18T04:30:31.093548+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-18T04:30:31.093548+00:00
[2022-11-18T04:30:32.611+0000] {base.py:71} INFO - Using connection ID 'spark_local' for task execution.
[2022-11-18T04:30:32.613+0000] {spark_submit.py:334} INFO - Spark-Submit cmd: spark-submit --master spark://Posathorns-MacBook-Pro-4.local:7077 --name arrow-spark json_to_spark.py
[2022-11-18T04:30:32.630+0000] {spark_submit.py:485} INFO - /opt/spark/bin/load-spark-env.sh: line 68: ps: command not found
[2022-11-18T04:30:32.631+0000] {spark_submit.py:485} INFO - /opt/spark/bin/spark-class: line 71: /usr/lib/jvm/java-11-openjdk-amd64//bin/java: No such file or directory
[2022-11-18T04:30:32.632+0000] {spark_submit.py:485} INFO - /opt/spark/bin/spark-class: line 96: CMD: bad array subscript
[2022-11-18T04:30:32.641+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 157, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 417, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://Posathorns-MacBook-Pro-4.local:7077 --name arrow-spark json_to_spark.py. Error code is: 1.
[2022-11-18T04:30:32.644+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=spark_operator, task_id=spark-job, execution_date=20221118T043031, start_date=20221118T043032, end_date=20221118T043032
[2022-11-18T04:30:32.652+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 35 for task spark-job (Cannot execute: spark-submit --master spark://Posathorns-MacBook-Pro-4.local:7077 --name arrow-spark json_to_spark.py. Error code is: 1.; 77)
[2022-11-18T04:30:32.698+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2022-11-18T04:30:32.742+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
